db/migrate/20170207155507_create_topics.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+class CreateTopics < ActiveRecord::Migration
+  def change
+    create_table :topics do |t|
+      t.string :slug, index: true
```

Comment:
why do we need a slug? topics aren't accesible via a controller, they don't need RESTfull  URIs